e2e/full_scan_test.go

```diff
@@ -180,6 +181,18 @@ var _ = ginkgo.Describe("Running a full scan (exploits, info finder, malware, mi
 				return *findings.Count > 0
 			}, DefaultTimeout*2, DefaultPeriod).Should(gomega.BeTrue())
 
+			if cfg.TestEnvConfig.Platform == testenvtypes.EnvironmentTypeDocker || cfg.TestEnvConfig.Platform == testenvtypes.EnvironmentTypeKubernetes {
+				ginkgo.By("checking if the scan has findings from different families")
+				gomega.Expect(*(*scans.Items)[0].Summary.TotalExploits).To(gomega.BeNumerically(">", 0), "expected exploits to be found")
```

Comment:
nit: we could make it a bit more cleaner to perform checks here
```suggestion
				ginkgo.By("checking if the scan has findings from different families")
				
				summary := *(*scans.Items)[0].Summary
				gomega.Expect(summary.TotalExploits).To(gomega.BeNumerically(">", 0), "expected exploits to be found")
```